db/orders.js

```diff
@@ -135,19 +134,77 @@ const createOrder = async (order) => {
             RETURNING *;
         `, values)
 
-        console.log(order)
-
         return order
     } catch (error) {
         throw error
     }
 }
 
+
+const updateOrder = async(orderToUpdate) => {
+    const {id} = orderToUpdate;
+    delete orderToUpdate.id;
+    const setString = Object.keys(orderToUpdate);
+    const changeStrings = setString.map((key, index) => {
+        return `"${key}" = $${index + 1}`
+    }).join(", ");
+
+    try {
+        const {rows: [order]} = await client.query(`
+        UPDATE orders
+        SET ${changeStrings}
+        WHERE id = ${id}
+        RETURNING *;
+        `, [...Object.values(orderToUpdate)]);
+
+        return order;
+
+    } catch (error) {
+        throw error;
+    }
+}
+
+
+
+const completeOrder = async (id) => {
+    try {
+        const {rows: [order]} = await client.query(`
```

Comment:
Maybe use `updateOrder` inside `completeOrder` and `cancelOrder`

Reply:
No thank you